docs/sapling-spending-key-encryption.md

```diff
@@ -0,0 +1,118 @@
+# Sapling Spending Key Encryption
```

Comment:
I wonder if the documentation is too low level for the general users. The users who want this level of detail could look at the code and unit test directly. 

On the website, we normally document the API provided by Taquito and how a user can use them rather than the implementation details. 
For example: The documentation could have live code examples showing how to instantiate the InMemorySpendingKey from a mnemonic, from a sk, from an encrypted sk. It could also describe the method available to the user; if the methods take parameters, and what they return.

Reply:
We can ask other team member opinion as well :)

Reply:
ok! i definitely blur the lines of high level and low level since i automatically assume everyone knows more than me 🙃

Reply:
I agree with Roxane, I think the vast majority of users only wants docs on how to use a certain thing. Contributors will dive into the code themselves